config/site/site.config.js

```diff
@@ -0,0 +1,122 @@
+const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
+const navConfig = require('./navigation.config');
+const startCase = require('lodash.startcase');
+const fs = require('fs');
+const path = require('path');
+
+const npmPackage = JSON.parse(fs.readFileSync(path.resolve(process.cwd(), 'package.json')));
+
+const siteConfig = {
+  /* The navigation configuration.  */
+  navConfig,
+
+  /* The path to the pages configuration. If this is enabled, page config will not be generated. */
+  // pagesConfig,
+
+  // These options are used to find the pages to serve via terra dev site.
+  // If 'pagesConfig' is provided, no pages will be generated.
+  // Output:
+  // {root}/lib/{entryPoint}/**/*{pageType}.{jsx,js,md,}
+  // {root}/packages/*/lib/{entryPoint}/**/*{pageType}.{jsx,js,md,}
+  // source will be used if hot reloading is enabled and not running in prod mode.
+  // customPatterns
+  // *NOTE* The entryPoints also act as the beginning of the directory strucure for menu navigation.
+  // For example: /root/src/<entryPointDir>/folder/item would start navigation at 'folder'.
+  generatePages: {
+    searchPatterns: [
```

Comment:
Is this an array such that we can indicate multiple roots and entry points in the search pattern? Or would you expect one root and multiple entry points?

Reply:
Multiple roots and entrypoints, the entrypoint is only applied to the root it's associated with.